cirq-ft/cirq_ft/algos/arithmetic_gates_test.py

```diff
@@ -322,3 +322,18 @@ def test_add_no_decompose(a, b):
     assert true_out_int == int(out_bin, 2)
     basis_map[input_int] = output_int
     cirq.testing.assert_equivalent_computational_basis_map(basis_map, circuit)
+
+
+@pytest.mark.parametrize("P,n", [(v, n) for n in range(1, 4) for v in range(1 << n)])
+@pytest.mark.parametrize("Q,m", [(v, n) for n in range(1, 4) for v in range(1 << n)])
+def test_decompose_less_than_equal_gate(P: int, n: int, Q: int, m: int):
```

Comment:
add a test for consistent t complexity

Reply:
`test_less_than_equal_consistent_protocols` already does this

Reply:
👍